cinema/models.py

```diff
@@ -35,12 +41,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def image_path(instance: "Movie", filename: str) -> pathlib.Path:
+    filename = (
+        f"{slugify(instance.title)}-{uuid.uuid4()}"
+        + pathlib.Path(filename).suffix
+    )
+    return pathlib.Path("upload/movies") / pathlib.Path(filename)
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(blank=True, upload_to=image_path)
```

Comment:
The use of `models.ImageField` for the `image` field in the `Movie` model is correct according to the checklist, which advises against using `serializers.CharField` for images.